src/coreclr/inc/corinfo.h

```diff
@@ -2733,7 +2737,10 @@ class ICorStaticInfo
                                CORINFO_FIELD_INFO    *pResult
                               ) = 0;
 
-    virtual uint32_t getThreadLocalFieldInfo (
+    virtual uint32_t getNonGCThreadLocalFieldInfo (
```

Comment:
Can you add function header comments here? What is the "Info" being returned? What is the `uint32_t` return value?

Reply:
Is it ok if I do it as part of my other PR? Don't want to run CI for comments.

Reply:
sure

Reply:
fixed this.